ivy_tests/test_ivy/test_frontends/test_numpy/test_linear_algebra/test_matrix_and_vector_products.py

```diff
@@ -197,3 +197,36 @@ def test_numpy_kron(
         a=xs[0],
         b=xs[1],
     )
+
+#multi_dot
+@handle_frontend_test(
+    fn_tree="numpy.linalg.multi_dot",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=5,
+        min_num_dims=2,
+        max_num_dims=2,
+        min_dim_size=2,
+        max_dim_size=2,
+        min_value=-100,
+        max_value=100,
+        shared_dtype=True,
+    )
+)
+def test_numpy_multi_dot(
+    dtype_and_x,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+):
+    dtypes, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=dtypes,
+        frontend=frontend,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        test_flags=test_flags,
+        arrays=(x[0], x[1], x[2], x[3], x[4]),
+        out=None,
+    )
```

Comment:
My suggested changes might introduce tolerance issues, I reccomend you add the following as well: 
`rtol=1e-3,`
`atol=1e-3,`